datafusion/sql/src/planner.rs

```diff
@@ -5992,6 +6011,20 @@ mod tests {
         quick_test(sql, expected);
     }
 
+    // TODO: will ad more tests to cover maby other cases
+    #[test]
+    fn test_prepare_statement_to_plan() {
+        let sql = "PREPARE my_plan(INT) AS SELECT id, age  FROM person WHERE age = $1";
+        //let statements = DFParser::parse_sql(sql).unwrap();
+
+        let expected = "Prepare: \"my_plan\" [Int32] \
+        \n  Projection: person.id, person.age\
+        \n    Filter: person.age = $1\
+        \n      TableScan: person";
+
+        quick_test(sql, expected);
+    }
+
```

Comment:
This is the only test for now. I will add more tests

Reply:
The other thing I don't see is code to figure out the placeholders and their types